build-support/bin/check_pants_pex_abi.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# Check that the ./pants.pex was built using the passed abi specification.
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+source ${REPO_ROOT}/build-support/common.sh
+CHECK_FOLDER="${REPO_ROOT}/pants_pex_abi_check"
+
+EXPECTED_ABI="$1"
+if [ -z "${EXPECTED_ABI}" ]; then
```

Comment:
Could there be a link the python abi docs either in a comment or in this message?